src/_data/sidenav.yml

```diff
@@ -237,8 +237,15 @@
           permalink: /docs/development/tools/hot-reload
         - title: Code formatting
           permalink: /docs/development/tools/formatting
-    - title: AndroidX migration
+    - title: Migration notes
       permalink: /docs/development/androidx-migration
+      children:
```

Comment:
See https://github.com/flutter/website/issues/3741#issuecomment-591822143.

@sfshaza2 Is this what you meant?  I copied `plugin-api-migration` into this section because it seems related.  I'm happy to revert all this if you'd like to rearrange to your liking.